simulations/vip-444/ethereum.ts

```diff
@@ -0,0 +1,131 @@
+        expect(price).to.be.eq(parseUnits("2898.719176821535543514", 18));
+      });
+    });
+
+    describe("PoolRegistry state", () => {
+      it(`should add ${market.vToken.symbol} to the Comptroller`, async () => {
+        const poolVTokens = await comptroller.getAllMarkets();
+        expect(poolVTokens).to.contain(market.vToken.address);
+      });
+
+      it(`should register ${market.vToken.symbol} in PoolRegistry`, async () => {
+        const registeredVToken = await poolRegistry.getVTokenForAsset(
+          COMPTROLLER_CORE,
+          market.vToken.underlying.address,
+        );
+        expect(registeredVToken).to.equal(market.vToken.address);
+      });
+    });
+
+    describe("Risk parameters", () => {
+      checkRiskParameters(market.vToken.address, market.vToken, market.riskParameters);
+
+      it("should pause borrowing on wsuperOETHb", async () => {
+        expect(await comptroller.actionPaused(market.vToken.address, 2)).to.equal(true);
+      });
+
+      it(`should have a protocol seize share ${market.riskParameters.protocolSeizeShare}`, async () => {
+        expect(await vTokenContract.protocolSeizeShareMantissa()).to.equal(market.riskParameters.protocolSeizeShare);
+      });
+    });
+
+    describe("Ownership and initial supply", () => {
+      const { vToken: vTokenSpec, initialSupply } = market;
+      const underlyingSymbol = token.symbol;
+
+      describe(`${vTokenSpec.symbol}`, () => {
+        it(`should have owner = normal timelock`, async () => {
+          expect(await vTokenContract.owner()).to.equal(NORMAL_TIMELOCK);
+        });
+
+        // Initial exchange rate should account for decimal transformations such that
+        // the string representation is the same (i.e. 1 vToken == 1 underlying)
+        const multiplier = 10 ** (vTokenSpec.underlying.decimals - vTokenSpec.decimals);
+        const vTokenSupply = initialSupply.amount.div(multiplier);
+        const underlyingSupplyString = formatUnits(initialSupply.amount, vTokenSpec.underlying.decimals);
+        const vTokenSupplyString = formatUnits(vTokenSupply, vTokenSpec.decimals);
+
+        it(`should have initial supply = ${vTokenSupplyString} ${vTokenSpec.symbol}`, async () => {
+          expect(await vTokenContract.balanceOf(initialSupply.vTokenReceiver)).to.equal(vTokenSupply);
+        });
+
+        it(`should have balance of underlying = ${underlyingSupplyString} ${underlyingSymbol}`, async () => {
+          const underlying = new ethers.Contract(vTokenSpec.underlying.address, ERC20_ABI, provider);
+          expect(await underlying.balanceOf(vTokenSpec.address)).to.equal(initialSupply.amount);
+        });
+      });
+    });
+
+    describe("Interest rates", () => {
+      checkInterestRate(
+        market.interestRateModel.address,
+        market.vToken.symbol,
+        market.interestRateModel,
+        BLOCKS_PER_YEAR,
+      );
+    });
+
+    checkIsolatedPoolsComptrollers();
+  });
```

Comment:
Maybe also check for conversion incentives

Reply:
https://github.com/VenusProtocol/vips/pull/484/commits/abc5bde91b713fd5fedc9b3c20fde55de85fef70